doc/geany.txt

```diff
@@ -2657,141 +2657,149 @@ Rarely used preferences, explained in the table below. A few of them require
+                                         reloading a file, and allow the operation
+                                         to be reverted.
+show_keep_edit_history_on_reload_msg     Whether to show a confirmation dialog to     true         immediately
+                                         drop the edit history on reloading a file,
+                                         see also `keep_edit_history_on_reload`
+                                         above.
+reload_clean_doc_on_file_change          Whether to automatically reload documents    false        immediately
+                                         that have no changes but which have changed
+                                         on disk.
+                                         If unsaved changes exist then the user is
+                                         prompted to reload manually.
+save_config_on_file_change               Automatically save Geany's configuration     true         immediately
+                                         to disk once the document list changes
+                                         (i.e. new documents are opened, saved or
+                                         closed). This helps to prevent accidentally
+                                         losing the session file list or other
+                                         changed settings when Geany is not shut
+                                         down cleanly. Disable this option if your
+                                         configuration directory is on a slow drive,
+                                         network share or similar and you experience
+                                         problems.
+extract_filetype_regex                   Regex to extract filetype name from file     See link     immediately
+                                         via capture group one.
+                                         See `ft_regex`_ for default.
 **``search`` group**
-find_selection_type               See `Find selection`_.                       0           immediately
-replace_and_find_by_default       Set ``Replace & Find`` button as default so  true        immediately
-                                  it will be activated when the Enter key is
-                                  pressed while one of the text fields has
-                                  focus.
+find_selection_type                      See `Find selection`_.                       0            immediately
+replace_and_find_by_default              Set ``Replace & Find`` button as default so  true         immediately
+                                         it will be activated when the Enter key is
+                                         pressed while one of the text fields has
+                                         focus.
+skip_confirmation_for_replace_in_session If set, do *not* show the                    false        immediately
```

Comment:
If it was lazy me I would have chosen a more cryptic but shorter name to avoid indenting the whole table, but since its done might as well leave it.

Reply:
Yeah, I don't like cryptic names anymore. I've chosen too many in the past and got annoyed myself even more often by them :).